hack/snapshot_to_catalog.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+
+usage() {
+  echo "Usage: $0 <snapshot-ref> <catalog-file> <bundle-tag>"
+  echo "  snapshot-ref: the snapshot reference to use, example: ols-bnxm2"
+  echo "  catalog-file: the catalog index file to update, example: lightspeed-catalog-4.16/index.yaml"
+  echo "  bundle-tag: the tag of the bundle to update, example: 0.1.0"
+  exit 1
```

Comment:
We will have the bundle-tag also in the snapshot right , we dont want the users to provide it 
Also we need to create seperate folders for all our konflux release catalog Dockerfile and the configs

Reply:
updated. we fetch bundle version from the OPM rendered bundle information.